contrib/zsh/_comma

```diff
@@ -0,0 +1,22 @@
+#compdef comma ,
+
+local -a args
+
+args=(
+  # Comma options
+  '(- 1 *)'{-h,--help}'[display usage information]'
+  '(- 1 *)'{-V,--version}"[show program's version number and exit]"
+  '(-i --install)'{-i,--install}'[install the derivation containing the executable]'
+  '(-s --shell)'{-s,--shell}'[open a shell containing the derivation containing the executable]'
+  '--picker[which picker to use when selecting a derivation]:<PICKER> [fzy]: _command_names -e'
+  '--nixpkgs_flake[which nixpkgs flake to use when selecting a derivation]:<NIXPKGS_FLAKE> [nixpkgs]'
```

Comment:
One missing feature would be to do something similar to [`nix-zsh-completions`](https://github.com/nix-community/nix-zsh-completions) and have suggestions of all flakes in the registry.